### PR TITLE
add daily summary to ForecastIO/weather

### PR DESCRIPTION
**Pull request by blockloop**
image

### CODE REVIEW

File: Weather/ForecastIO/weather.15m.py [hunk 1]
```diff
@@ -201,6 +207,10 @@ def render_wx():
 
   print '---'
 
+  if 'pressure' in weather_data:
```
Comment (seripap) [inline Weather/ForecastIO/weather.15m.py]: This is pulling the wrong key value. I would also wrap this below the radar image and keep the next hour summary at the top as it's the most relevant.

Something like this:

```
  print '---'

  if 'next_hour' in weather_data:
    print weather_data['next_hour']

  print '---'

  if 'image' in weather_data:
    if weather_data['image']['encoded']:
      print '| href=' + weather_data['image']['url'] + '| image=' + weather_data['image']['encoded'] 

  print '---'

  if 'week' in weather_data:
    print "\n".join(textwrap.wrap(weather_data['week'], 50))
     print '---'
```

**Comment by seripap** [comment]
@blockloop could you fix from the reference in the diff? Should be good afterwards. Thanks!

**Comment by blockloop** [comment]
oops :) fixed it

File: Weather/ForecastIO/weather.15m.py [hunk 2]
```diff
@@ -201,6 +207,10 @@ def render_wx():
 
   print '---'
 
+  if 'week' in weather_data:
+    print "\n".join(textwrap.wrap(weather_data['week'], 50))
+    print '---'
+
   if 'next_hour' in weather_data:
```
Comment (seripap) [inline Weather/ForecastIO/weather.15m.py]: Can you move this block to line 203, above the image and add another line break? This information is more relevant versus the weekly summary so it should naturally flow as you're reading

Comment (blockloop) [inline Weather/ForecastIO/weather.15m.py]: Like this?

image

**Comment by blockloop** [comment]
I moved the daily summary above the image

**Comment by seripap** [comment]
@blockloop the current summary should be up there- "Clear for the hour'. Swap the with the daily summary

**Comment by blockloop** [comment]
I moved it, but it wasn't there before I made my changes. Hourly was after the image.

**Comment by seripap** [comment]
Great- with the addition of daily, it made sense to move it above the image; since the image shows what is actually going on in terms of precipitation for the next hour. 

@keithamus good to merge 👍

**Comment by tresni** [comment]
Thanks for the discussion and PR!

**Pull request merged**